src/fmeval/model_runners/extractors/jumpstart_extractor.py

```diff
@@ -56,15 +56,19 @@ def __init__(
             self._sagemaker_session.boto_region_name,
         )
         util.require(DEFAULT_PAYLOADS in model_spec_key, f"Model: {jumpstart_model_id} is not supported at this time")
-        output_jmespath_expressions = compile_jmespath(GENERATED_TEXT_JMESPATH_EXPRESSION).search(
-            model_spec_key[DEFAULT_PAYLOADS]
-        )
-        util.require(
-            output_jmespath_expressions,
-            f"Jumpstart model {jumpstart_model_id} is likely not supported. If you know it is generates text, "
-            f"please provide output and log_probability jmespaths to the JumpStartModelRunner",
-        )
-        self._output_jmespath_compiler = compile_jmespath(output_jmespath_expressions[0])
+
+        try:
+            output_jmespath_expressions = compile_jmespath(GENERATED_TEXT_JMESPATH_EXPRESSION).search(
+                model_spec_key[DEFAULT_PAYLOADS]
+            )
+            util.assert_condition(
+                output_jmespath_expressions,
+                f"Jumpstart model {jumpstart_model_id} is likely not supported. If you know it is generates text, "
+                f"please provide output and log_probability jmespaths to the JumpStartModelRunner",
+            )
+            self._output_jmespath_compiler = compile_jmespath(output_jmespath_expressions[0])
+        except EvalAlgorithmClientError as e:
```

Comment:
does this come from assert_condition? Let's throw internal error directly instead of mutating it.

Reply:
`assert_condition` throws `EvalAlgorithmInternalError`. The error we are catching `EvalAlgorithmClientError` comes from `compile_jmespath` - this is used in other places where it is indeed a client error.